README.md

```diff
@@ -47,6 +47,36 @@ use its id instead.
 See the [Sidekiq best practices](https://github.com/mperham/sidekiq/wiki/Best-Practices)
 for more information.
 
+### YAML permitted classes
+
+The gem internally uses YAML serialization/deserialization to pass your event data and subscribers to a sidekiq worker through redis.
+
+By default, for security reasons, only a few basic classes like `String` or `Array` are permitted for deserialization.
+
+If you are using custom types as your event data, you might run into
+```
+Psych::DisallowedClass:
+   Tried to load unspecified class: MyEventData1
+```
+
+In that case, you can explicitly allow custom types using
+
+```ruby
+Wisper::Sidekiq.configure do |config|
+  config.register_safe_types(MyEventData1, MyEventData2)
+end
```

Comment:
is this needed, as it looks like types are auto-registered, or would I need to do something to have the auto-registration enabled?